Author: Simon Horman <horms@debian.org>
Description: New magic for the Ulead Photo Explorer5 image format (Closes: #339397).

diff -Naurp file.org/magic/Magdir/images file/magic/Magdir/images
--- file.org/magic/Magdir/images	2009-02-02 15:55:49.000000000 +0000
+++ file/magic/Magdir/images	2009-02-18 11:28:44.000000000 +0000
@@ -605,3 +605,8 @@
 # JPEG 2000 Code Stream Bitmap
 # From Petr Splichal <psplicha@redhat.com>
 0	string	\xFF\x4F\xFF\x51\x00	JPEG-2000 Code Stream Bitmap data
+
+# Type:	Ulead Photo Explorer5 (.pe5)
+# URL:	http://www.jisyo.com/cgibin/view.cgi?EXT=pe5 (Japanese)
+# From:	Simon Horman <horms@debian.org>
+0	string	IIO2H			Ulead Photo Explorer5
